earth_enterprise/src/fusion/fusionui/rasterimportbase.ui

```diff
@@ -852,7 +852,7 @@
     </connection>
 </connections>
 <tabstops>
-    <tabstop>aquisitionDate</tabstop>
+    <tabstop>acquisitionDate</tabstop>
```

Comment:
Since you capitalized AcquisitionDate above, we should capitalize it here as well.